File: Tema2/Ejercicio2/app/src/main/java/com/example/ejercicio2/MainActivity.java
package com.example.ejercicio2;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.Menu;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button player = (Button) findViewById(R.id.button2);
        player.setOnClickListener(v -> launchNewPlayer());

    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        getMenuInflater().inflate(R.menu.main, menu);
        return super.onCreateOptionsMenu(menu);
    }

    private void launchNewPlayer(){
        Intent i = new Intent(this, NewPlayer.class);
        startActivity(i);
    }
}

File: Tema2/DIEjercicio1/app/src/main/java/com/example/diejercicio1/MainActivity.kt
package com.example.myapplication

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import com.example.diejercicio1.R

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val player = findViewById(R.id.button2) as Button
        player.setOnClickListener{launchNewPlayer()}
    }

    fun launchNewPlayer(){
        val i = Intent(this, NewPlayer::class.java)
        startActivity(i)
    }
}

